pkg/csi/service/mounter/mounter_windows.go

```diff
@@ -148,59 +148,58 @@ func normalizeWindowsPath(path string) string {
 }
 
 // ExistsPath - Checks if a path exists. This call does not perform follow link.
-func (mounter *csiProxyMounter) ExistsPath(path string) (bool, error) {
-	ctx := mounter.Ctx
+func (mounter *csiProxyMounter) ExistsPath(ctx context.Context, path string) (bool, error) {
 	log := logger.GetLogger(ctx)
-	isExistsResponse, err := mounter.FsClient.PathExists(context.Background(),
+	isExistsResponse, err := mounter.FsClient.PathExists(ctx,
 		&fs.PathExistsRequest{
 			Path: normalizeWindowsPath(path),
 		})
 	if err != nil {
-		log.Errorf("Proxy returned error while checking if PathExists: %v", err)
+		log.Errorf("Proxy returned error while checking if PathExists for path: %q, err: %v", path, err)
 		return false, err
 	}
 	return isExistsResponse.Exists, err
 }
 
 // Rmdir - delete the given directory
-func (mounter *csiProxyMounter) Rmdir(path string) error {
+func (mounter *csiProxyMounter) Rmdir(ctx context.Context, path string) error {
+	log := logger.GetLogger(ctx)
 	rmdirRequest := &fs.RmdirRequest{
 		Path:  normalizeWindowsPath(path),
 		Force: true,
 	}
 	_, err := mounter.FsClient.Rmdir(context.Background(), rmdirRequest)
 	if err != nil {
+		log.Errorf("failed to remove dir: %q with force. err: %v", rmdirRequest.Path, err)
 		return err
 	}
 	return nil
 }
 
 // MakeDir - Creates a directory.
-func (mounter *csiProxyMounter) MakeDir(pathname string) error {
-	ctx := mounter.Ctx
+func (mounter *csiProxyMounter) MakeDir(ctx context.Context, pathname string) error {
 	log := logger.GetLogger(ctx)
 	mkdirReq := &fs.MkdirRequest{
 		Path: normalizeWindowsPath(pathname),
 	}
-	_, err := mounter.FsClient.Mkdir(context.Background(), mkdirReq)
+	_, err := mounter.FsClient.Mkdir(ctx, mkdirReq)
 	if err != nil {
-		log.Infof("Error: %v", err)
+		log.Errorf("failed to make directory: %q. Error: %v", mkdirReq.Path, err)
 		return err
 	}
 	return nil
 }
 
 // Gets windows specific disk number from diskId
-func (mounter *csiProxyMounter) GetDiskNumber(diskID string) (string, error) {
-	ctx := mounter.Ctx
+func (mounter *csiProxyMounter) GetDiskNumber(ctx context.Context, diskID string) (string, error) {
 	log := logger.GetLogger(ctx)
 	// Check device is attached
 	log.Debug("GetDiskNumber called with diskID: %q", diskID)
 
 	listRequest := &disk.ListDiskIDsRequest{}
-	diskIDsResponse, err := mounter.DiskClient.ListDiskIDs(context.Background(), listRequest)
+	diskIDsResponse, err := mounter.DiskClient.ListDiskIDs(ctx, listRequest)
 	if err != nil {
-		log.Debug("Could not get diskids %s", err)
+		log.Debug("Could not get diskids. error: %v", err)
```

Comment:
Should be Debugf ?

This can even be Errorf

Reply:
fixed.